cpu/kinetis/periph/uart.c

```diff
@@ -63,6 +63,15 @@
 #define LPUART_OVERSAMPLING_RATE (16)
 #endif
 
+#ifndef LPUART_0_SRC
+/* Default LPUART clock setting to avoid compilation failures, define this in
+ * periph_conf.h to set board specific configuration if using the LPUART. */
```

Comment:
Maybe this comment can be above the definition?

Reply:
addressed